django_fido/static/django_fido/js/fido2.js

```diff
@@ -62,6 +65,10 @@ function fido2SuccessAuthenticationCallback(assertion) {
 
 function fido2ErrorResponseCallback(error) {
     addFido2Error(error.message);
+    document.getElementById('submit-button').style.opacity = "1";
```

Comment:
I'd remove this style change. It should be part the particular `closeFidoWindow` hook.

Reply:
Removed